src/com/github/jojo2357/RepoRefiner.java

```diff
@@ -0,0 +1,31 @@
+package com.github.jojo2357;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class RepoRefiner {
+    public static void main(String[] args) throws IOException {
```

Comment:
remove throws (i can do this, just making a note)

Reply:
aight i guess ill remove throws

Reply:
i can do it dw